model/transaction/_meta/fields.yml

```diff
@@ -16,6 +16,17 @@
               description: >
                 Total duration of this transaction, in microseconds.
 
+            - name: count
+              type: long
+              count: 1
```

Comment:
i couldn't find what is this `count`, just copied

Reply:
I think you can safely remove them from these field mappings.
If I recall correctly they are written to the Kibana Index Pattern in the index-pattern/template creation process and we only added it for some fields where it was necessary for the dashboards and visualizations.